docs/src/main/asciidoc/quartz.adoc

```diff
@@ -133,10 +133,20 @@ public class TaskBean {
     }
 }
 ----
+
+[source,conf]
+----
+quarkus.quartz.misfire-policy.task-job=ignore-misfire-policy <5>
+----
```

Comment:
Maybe this can go in this part https://github.com/quarkusio/quarkus/blob/3f1926ae23d4ea1276669b0f8dd960d73feee6ee/docs/src/main/asciidoc/quartz.adoc#updating-the-application-configuration-file